Package@swift-5.4.swift

```diff
@@ -0,0 +1,48 @@
+// swift-tools-version:5.3
```

Comment:
Should this be 5.4?

Reply:
In the past, I've used older Swift tools versions in version-specific manifests (e.g. `// swift-tools-version:5.1` for Swift 5.3), so I think this should work. If it doesn't for any reason, let me know and I'll update it.